.github/workflows/rust.yml

```diff
@@ -37,21 +37,21 @@ jobs:
     container:
       image: ${{ matrix.docker_image }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
 
     - name: Search packages in this repository
       id: list_packages
       run: |
         echo ::set-output name=package_list::$(colcon list --names-only)
 
     - name: Setup ROS environment
-      uses: ros-tooling/setup-ros@v0.6
+      uses: ros-tooling/setup-ros@v0.7
       with:
         required-ros-distributions: ${{ matrix.ros_distribution }}
         use-ros2-testing: ${{ matrix.ros_distribution == 'rolling' }}
 
     - name: Setup Rust
-      uses: dtolnay/rust-toolchain@1.71.0
+      uses: dtolnay/rust-toolchain@1.74.0
```

Comment:
Update Dockerfile as well?

Reply:
Done in be4225a2d7adf9fe66ef336374c38dd941128bea